### PR TITLE
create squirrel_waypoint_msgs, added ExamineWaypoint.srv

### PR DESCRIPTION
**Pull request by edith-langer**

### CODE REVIEW

File: squirrel_waypoint_msgs/package.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <name>squirrel_waypoint_msgs</name>
+  <version>0.0.5</version>
+  <description>Messages, actions and services for creating waypoints for the planner</description>
+  <maintainer email="langer@acin.tuwien.ac.at">Edith Langer</maintainer>
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>actionlib</build_depend>
```
Comment (NadiaHG) [inline squirrel_waypoint_msgs/package.xml]: why do you need actionlib and actionlib_msgs?

File: squirrel_waypoint_msgs/package.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <build_depend>actionlib_msgs</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>message_generation</build_depend>
+  <build_depend>std_msgs</build_depend>
+
+  <run_depend>actionlib</run_depend>
+  <run_depend>actionlib_msgs</run_depend>
+  <run_depend>geometry_msgs</run_depend>
+  <run_depend>message_runtime</run_depend>
+  <run_depend>rospy</run_depend>
```
Comment (NadiaHG) [inline squirrel_waypoint_msgs/package.xml]: do you need rospy?

**Comment by ipa-nhg** [comment]
please add the package to the metapackage package.xml: https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/squirrel_common/package.xml

File: squirrel_waypoint_msgs/package.xml (hunk 1 above)
Comment (edith-langer) [inline squirrel_waypoint_msgs/package.xml]: Removed it!

File: squirrel_waypoint_msgs/package.xml (hunk 2 above)
Comment (edith-langer) [inline squirrel_waypoint_msgs/package.xml]: No, I deleted it

**Comment by edith-langer** [comment]
I added squirrel_waypoint_msgs to the metapackage

**Comment by ipa-nhg** [comment]
Thanks @edith-langer 

LGTM :+1:

**Pull request merged**